week_2/ec2.tf

```diff
@@ -0,0 +1,16 @@
+variable "instance_name" {
+  description = "Value of the Name tag for the EC2 instance"
+  type        = string
+  default     = "ExampleAppServerInstance"
+}
+
+resource "aws_instance" "app_server" {
```

Comment:
> o	Implement file downloading from S3 bucket during instance startup (bucket and file should be created by init-s3.sh script).

- `init-s3.sh` should also be present in this PR
- And instance startup logic for this instance

Reply:
- added key-name for ssh connection
- changed s3 access to getObject only
- added "user_data"  for getting file from s3 during